.github/workflows/conda-build.yml

```diff
@@ -0,0 +1,61 @@
+name: conda-build
+on:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+
+jobs:
+  conda-build:
+    name: ${{ matrix.conda_build_yml }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { conda_build_yml: linux_64_python3.7_default.____cpython,  os: ubuntu-latest,  conda-build-args: '' }
+          - { conda_build_yml: linux_64_python3.8_default.____cpython,  os: ubuntu-latest,  conda-build-args: '' }
+          - { conda_build_yml: linux_64_python3.9_default.____cpython,  os: ubuntu-latest,  conda-build-args: '' }
+          - { conda_build_yml: linux_64_python3.10_default.____cpython, os: ubuntu-latest,  conda-build-args: '' }
+          - { conda_build_yml: osx_64_python3.7_default.____cpython,    os: macos-latest,   conda-build-args: '' }
+          - { conda_build_yml: osx_64_python3.8_default.____cpython,    os: macos-latest,   conda-build-args: '' }
+          - { conda_build_yml: osx_arm64_python3.9.____cpython,         os: macos-latest,   conda-build-args: ' --no-test' }
+          - { conda_build_yml: osx_arm64_python3.10.____cpython,        os: macos-latest,   conda-build-args: ' --no-test' }
+          - { conda_build_yml: win_64_python3.9.____cpython,            os: windows-latest, conda-build-args: '' }
+          - { conda_build_yml: win_64_python3.10.____cpython,           os: windows-latest, conda-build-args: '' }
```

Comment:
This is copied from the various conda build workflows. Not all variants are tested in CI, like `linux_aarch64_python3.10_default.____cpython`. Should we also test aarch64 and ppc64le in CI?

Reply:
I'll try it out.

Reply:
~6min seems fine. Quicker than macOS and Windows 😅

Reply:
Yes, please add them.